cli/configuration/v1/config.go

```diff
@@ -0,0 +1,45 @@
+package v1
+
+import (
+	"encoding/json"
+
+	"ocm.software/open-component-model/bindings/go/runtime"
+)
+
+var scheme = runtime.NewScheme()
+
+func init() {
+	scheme.MustRegisterWithAlias(&Config{}, runtime.NewUngroupedVersionedType(ConfigType, ConfigTypeV1))
+}
+
+const (
+	ConfigType   = "generic.config.ocm.software"
+	ConfigTypeV1 = "v1"
+)
+
+// Config holds configuration entities loaded through a configuration file.
+// +k8s:deepcopy-gen=true
```

Comment:
@jakobmoellerdev How do you run the generate workflow? :D It keeps erroring with weird errors like, module not found, no main module, etc...

Reply:
I think your problem is that youre generating types in the CLI. The code generator is currently setup to only run on go modules. Check the tools section for that

Reply:
After you included the cli, it should just be `task generate`

Reply:
I tried that but it gave me different errors. Let me do that again.